packages/design-system-react/tsconfig.json

```
@@ -9,7 +9,10 @@
       "@testing-library/react",
       "@testing-library/jest-dom"
     ],
-    "lib": ["ES2020", "DOM"]
+    "lib": ["ES2020", "DOM"],
+    "module": "ESNext",
+    "moduleResolution": "node",
+    "esModuleInterop": true
```

We can revert this it was a storybook issue not an issue with the design-system-react package phew!
```suggestion
    "lib": ["ES2020", "DOM"],
```